classes/issue/IssueDAO.inc.php

```diff
@@ -302,8 +302,18 @@ function _returnIssueFromRow($row) {
 
 		$this->getDataObjectSettings('issue_settings', 'issue_id', $row['issue_id'], $issue);
 
-		HookRegistry::call('IssueDAO::_returnIssueFromRow', array(&$issue, &$row));
+		HookRegistry::call('IssueDAO::_fromRow', array(&$issue, &$row));
+
+		return $issue;
+	}
 
+	/**
+	 * @copydoc self::_fromRow()
+	 * @deprecated
```

Comment:
(It would be nice to add a date here, so later greps for deprecation can give an easy sense of whether it's reasonable to remove something)

Reply:
:+1: